CHANGELOG.md

```diff
@@ -23,6 +23,28 @@ Classify the change according to the following categories:
     ### Deprecated
     ### Removed
 
+## v0.47.0
```

Comment:
just noting there is a version conflict between the changelog and the manifest.toml

Reply:
Also one other note: `ProcessHeatLoad` was already part of the v46.0 update, so perhaps update that to `BuiltInProcessHeatLoad`?

Reply:
@zolanaj @Bill-Becker , let me know if there is anything i can do to help fix these inconsistencies. Since I can't change it directly on dev, I can start a branch if necessary to update the Chagelog

Reply:
I think I removed all of the redundant ones from v0.46.0 and replaced with a general description of what was added for ProcessHeatLoad, to supplement the other bullet points in the Updated section.